util/options.py

```diff
@@ -207,7 +208,7 @@ def __init__(
         :param read_experiments_from_images: Try to read the experiments from images
         :param check_format: Check the format when reading images
         :param verbose: True/False print out some stuff
-
+        :param load_models: whether to load models from every experiment
```

Comment:
Since this just passes through to `try_read_experiments_from_images`, it should have the same description. At the moment it's different.